editor/blocks/quote-block/index.js

```diff
@@ -0,0 +1,23 @@
+const { query, html, text } = wp.blocks.query;
+
+wp.blocks.registerBlock( 'wp/quote', {
+	title: 'Quote',
+	icon: 'format-quote',
+
+	attributes: {
+		value: query( 'blockquote', html( 'p' ) ),
+		citation: query( 'cite,footer', text() )
+	},
+
+	edit( attributes, onChange ) {
+		return wp.element.createElement( wp.blocks.EditableQuote, {
+			value: attributes.value,
+			citation: attributes.citation,
+			onChange: ( value ) => onChange( { value } )
+		} );
+	},
+
+	save( attributes ) {
+		return wp.element.createElement( 'p', null, attributes.value );
```

Comment:
I figured, I was waiting on feedback before going ahead and changing this.

>  the responsibility of the save function to return the correct markup to be stored in postContent

Should it use `renderToString(element)` instead as a return value?  Or what is the purpose of returning the react element itself?